res/openvpn-gui-res-pl.rc

```diff
@@ -531,18 +532,18 @@ Prawdopodobnie OpenVPN nie jest zainstalowany."
 z prawami administratora aby uaktualnić rejestr."
     IDS_ERR_READ_SET_KEY "Błąd odczytu/zmiany klucza rejestru ""%ls""."
     IDS_ERR_WRITE_REGVALUE "Błąd zapisu rejestru ""HKEY_CURRENT_USER\\%ls\\%ls""."
-    IDS_ERR_PLAP_REG "Failed to enable Start Before Logon (error = %d)"
-    IDS_ERR_PLAP_UNREG "Failed to disable Start Before Logon (error = %d)"
+    IDS_ERR_PLAP_REG "Błąd podczas włączania łączenia przed logowaniem (błąd = %d)"
+    IDS_ERR_PLAP_UNREG "Błąd podczas wyłączania łączenia przed logowaniem (błąd = %d)"
```

Comment:
IDS_ERR_PLAP_UNREG "Błąd podczas zakańczania połączenia przed logowaniem (błąd = %d)"

Reply:
This is the same problem as before: it is not about a certain connection.